src/main/java/neko/Storage.java

```diff
@@ -0,0 +1,82 @@
+package neko;
+import neko.task.*;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.io.File;
+import java.util.Scanner;
+import java.io.FileWriter;
+
+public class Storage {
+
+    private static final String DIRECTORY_PATH = "./data";
```

Comment:
I like how you made DIRECTORY_PATH a constant. Any particular reason why this is hardcoded and not configurable? Should we consider making it flexible for future changes?